docs/maps/vector-style.asciidoc

```diff
@@ -45,11 +45,17 @@ image::maps/images/vector_style_dynamic.png[]
 
 Quantitative data driven styling symbolizes features from a range of numeric property values.
 
-To ensure symbols are consistent as you pan, zoom, and filter the map, quantitative data driven styling uses {ref}/search-aggregations-metrics-extendedstats-aggregation.html[extended_stats aggregation] to retrieve statistical metadata.
+Property values are fit from the domain range to the style range on a linear scale.
```

Comment:
Not sure what this sentence means.  Also needs to flow better with the previous sentence.